src/fdk_rdf_parser/classes/opening_hours_specification.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+
+@dataclass
+class OpeningHoursSpecification:
+    uri: Optional[str] = None
+    dayOfWeek: Optional[List[str]] = None
+    opens: Optional[str] = None
+    closes: Optional[str] = None
+    validFrom: Optional[str] = None
+    validThrough: Optional[str] = None
```

Comment:
Samme her som over, `CVOpeningHoursSpecification`.

Reply:
Har i hovedsak kun tatt med prefiksene når det er potensial for like navn, tror ikke det er så mange andre som kommer til å bruke navnet `openingHoursSpecification`

Reply:
Kan ta det med til senere, slik at vi får gjort konsekvent over alt.